README.md

```diff
@@ -1,87 +1,81 @@
-# wavefront-lambda-go [![travis build status](https://travis-ci.com/wavefrontHQ/wavefront-lambda-go.svg?branch=master)](https://travis-ci.com/wavefrontHQ/wavefront-lambda-go)
+# wavefront-lambda-go
 
-This is a Wavefront Go wrapper for AWS Lambda to enable reporting standard lambda metrics and custom app metrics directly to wavefront.
+[![travis build status](https://travis-ci.com/wavefrontHQ/wavefront-lambda-go.svg?branch=master)](https://travis-ci.com/wavefrontHQ/wavefront-lambda-go)
+[![Go Report Card](https://goreportcard.com/badge/github.com/wavefrontHQ/wavefront-lambda-go)](https://goreportcard.com/report/github.com/wavefrontHQ/wavefront-lambda-go)
 
-## Requirements
-Go 1.x
+A Go wrapper for AWS Lambda so you can monitor everything from your [Wavefront](https://wavefront.com) dashboard
 
 ## Installation
-```
+
+Using `go get`
+
+```bash
 go get github.com/wavefronthq/wavefront-lambda-go
 ```
 
-## Environment variables
-WAVEFRONT_URL = https://\<INSTANCE>.wavefront.com  
-WAVEFRONT_API_TOKEN = Wavefront API token with Direct Data Ingestion permission.  
-REPORT_STANDARD_METRICS = Set to False or false to not report standard lambda metrics directly to wavefront.  
-
 ## Usage
 
-Wrap your AWS Lambda handler function with wavefront_lambda.Wrapper(LambdaHandler).
+To connect your Lambda functions to Wavefront, you'll need to set two environment variables, import this module, and wrap your AWS Lambda handler function with `wavefront_lambda.Wrapper(LambdaHandler)`. The environment variables you'll need to set are:
+
+* `WAVEFRONT_URL`: The URL of your Wavefront instance (like, `https://myinstance.wavefront.com`).
+* `WAVEFRONT_API_TOKEN`: Your Wavefront API token (see the [docs](https://docs.wavefront.com/wavefront_api.html) how to create an API token).
 
 ```go
 package main
 
 import (
+	"github.com/aws/aws-lambda-go/events"
 	"github.com/aws/aws-lambda-go/lambda"
-	"github.com/rcrowley/go-metrics"
-	"github.com/wavefronthq/go-metrics-wavefront"
-	"github.com/wavefronthq/wavefront-lambda-go"
+	wflambda "github.com/wavefronthq/wavefront-lambda-go"
 )
 
-// Lambda handler function that includes the code which will be executed when lambda is invoked.
-func HandleLambdaRequest() {
-	// your code
+func handler() (string, error){
+	return "Hello World", nil
 }
 
 func main() {
-	// Wrap your Lambda Handler Function with wflambda.Wrapper
 	lambda.Start(wflambda.Wrapper(HandleLambdaRequest))
```

Comment:
Just trying to understand why remove comments? 
if you do want to keep this change this line should be wflambda.Wrapper(handler)

Reply:
I've added comments in the first example, and updated the function call in `main`